### ISSUE TITLE
multi-gpu batch normalization

### ISSUE THREAD

**Issue by dongzhuoyao** [dongzhuoyao/mxnetgo#9]
RT

**Comment by dongzhuoyao** [comment]
https://mxnet.incubator.apache.org/api/python/symbol.html#mxnet.symbol.BatchNorm

**Comment by dongzhuoyao** [comment]
https://zhuanlan.zhihu.com/p/27069202
